packages/amplify-cli/src/extensions/amplify-helpers/resource-status-view.ts

```diff
@@ -0,0 +1,36 @@
+import { getSummaryTableData, getResourceDiffs } from './resource-status-data';
+import * as resourceStatus from './resource-status-diff';
+import { getEnvInfo } from './get-env-info';
+import { print } from './print';
+import chalk from 'chalk';
+//view: displays resource-diff (cloudformation-diff, input parameters (pending))
+export async function viewResourceDiffs({ resourcesToBeUpdated, resourcesToBeDeleted, resourcesToBeCreated }) {
+  const resourceDiffs = await getResourceDiffs(resourcesToBeUpdated, resourcesToBeDeleted, resourcesToBeCreated);
+  for await (let resourceDiff of resourceDiffs.updatedDiff) {
```

Comment:
```suggestion
  for await (const resourceDiff of resourceDiffs.updatedDiff) {
```

Also for other loops